library/ecp.c

```diff
@@ -2526,7 +2526,7 @@ static int ecp_mul_mxz( mbedtls_ecp_group *grp, mbedtls_ecp_point *R,
     MBEDTLS_MPI_CHK( ecp_randomize_mxz( grp, &RP, f_rng, p_rng ) );
 
     /* Loop invariant: R = result so far, RP = R + P */
-    i = mbedtls_mpi_bitlen( m ); /* one past the (zero-based) most significant bit */
```

Comment:
I'm happy with the change, but it introduces an unwritten/unchecked requirement that the length of m <= length of the group. I think this should be documented at the top of the function, in case we wanted to call this function in some other way in future.

Reply:
Not sure this minor point justifies holding this up.